tests/solana_tests/yul.rs

```diff
@@ -502,7 +502,10 @@ contract testing  {
     runtime.constructor(&[]);
     let returns = runtime.function("test_address", &[]).unwrap();
     let addr = returns.into_bigint().unwrap();
-    let b_vec = addr.to_bytes_be().1;
+    let mut b_vec = addr.to_bytes_be().1;
+    while b_vec.len() < 32 {
+        b_vec.insert(0, 0);
+    }
```

Comment:
Why is this necessary?

Reply:
```rust
    let returns = runtime.function("test_address", &[]).unwrap();
    let addr = returns.into_bigint().unwrap();
    let b_vec = addr.to_bytes_be().1;
    let mut b_vec = addr.to_bytes_be().1;
```
The function returns `address` as `uint256`. If the highest bits are 0, then `addr.to_bytes_be().1` may not return 32 bytes.

Reply:
I'll add a comment